identity/aziot-identityd/src/identity.rs

```diff
@@ -736,15 +751,33 @@ impl IdentityManager {
         &self,
         identity_pk: &str,
         identity_cert: &str,
-        subject: &str,
-    ) -> Result<(), Error> {
+        subject: Option<&str>,
```

Comment:
This should be `Option<String>`, since `registration_id` in the config is already `Option<String>`.

Reply:
Fixed